src/frontend/pages/playlist/PlaylistFolder.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useAuthContext, useActionContext } from "../../context";
+import { Link } from "react-router-dom";
```

Comment:
You can use `import { Link, useParams } from "react-router-dom";`  instead of importing them separately.

Reply:
Thanks for Pointing  this out will do the changes

Reply:
done